webui/react/src/services/decoder.ts

```diff
@@ -220,6 +220,7 @@ export const ioToExperimentConfig =
     description: io.description,
     hyperparameters: ioToExperimentHyperparameters(io.hyperparameters),
     labels: io.labels || undefined,
+    profiling: { enabled: io?.profiling?.enabled === true },
```

Comment:
**nit:** haven't tested this but this should work?
```
profiling: { enabled: !!io.profiling?.enabled },
```
- `?` after `io` shouldn't be needed
- double bang to evaluate to a boolean (down side is if `enabled` is anything other than undefined, null, 0, empty string it will be true, but should be perfectly fine here)

Reply:
just saw in `ioTypes` that it has to be boolean or undefined so double bang will for sure work